kit/src/main/java/com/oracle/javafx/scenebuilder/kit/skeleton/SkeletonCreatorKotlin.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) 2016, 2021, Gluon and/or its affiliates.
+ * Copyright (c) 2012, 2014, Oracle and/or its affiliates.
```

Comment:
New files doesn't need entry for Oracle. Check [AbstractSkeletonCreator.java](https://github.com/gluonhq/scenebuilder/pull/328/files#diff-0d03d25c4354d83495020d140ffb31145769d5538a5a3a40420351376a4f4356R2) for correct license header.